docs/_docs/reference/new-types/union-types.md

```diff
@@ -22,7 +23,10 @@ Union types are duals of intersection types. `|` is _commutative_:
 `A | B` is the same type as `B | A`.
 
 The compiler will assign a union type to an expression only if such a
-type is explicitly given. This can be seen in the following [REPL](https://docs.scala-lang.org/overviews/repl/overview.html) transcript:
+type is explicitly given or the common supertype of all alternatives is [transparent](../other-new-features/transparent-traits.md).
```

Comment:
I think the sentence would be a bit easier to parse with an extra if:
```suggestion
type is explicitly given or if the common supertype of all alternatives is [transparent](../other-new-features/transparent-traits.md).
```
Otherwise the updated doc LGTM.